### ISSUE TITLE
Train

### ISSUE THREAD

**Issue by yohehe** [yohehe/pytorch_temp#3]

**Comment by yohehe** [comment]
`
# DataLoaderの基本構造

from torch.utils.data import TensorDataset,DataLoader

# Data Frameから読み取って処理していくDataSet例

class My_Dataset(Dataset):
    
    def __init__(self, train_data,target_data):
        self.data = train_data
        self.target= target_data
        #self.transform = transform
        
    def __len__(self):
        return len(self.data)
    
    def __getitem__(self, index):
        # load image as ndarray type (Height * Width * Channels)
        # be carefull for converting dtype to np.uint8 [Unsigned integer (0 to 255)]
        # in this example, i don't use ToTensor() method of torchvision.transforms
        # so you can convert numpy ndarray shape to tensor in PyTorch (H, W, C) --> (C, H, W)
        #image = self.data.iloc[index, 1:].values.astype(np.uint8).reshape((1, 28, 28))
        #label = self.data.iloc[index, 0]
        
        data=self.data.iloc[index]
        data=data.to_numpy()
        
        target=self.target.iloc[index]
        target=target.to_numpy()
        #if self.transform is not None:
            #image = self.transform(image)
            
        #to tensor
        data=data.astype(np.float32)
        data=torch.from_numpy(data)
        
        target=target.astype(np.float32)
        target=torch.from_numpy(target)
            
        return data,target
`
